byway/src/main/webapp/track-interests.js

```diff
@@ -0,0 +1,55 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+let selected = [];
+
+/** 
+ * Track the selected interests in an array according to their status. 
+ * @param elem tracks the current button element chosen.
+ */
+function checkStatus(elem) {
+  if(elem.className === "btnActive") {
+    selected.push(elem.innerText);
+  } else {
+    selected = selected.filter(function(interest) {
+      return interest !== elem.innerText;
+    });
+  }
+}
+
+/** 
+ * Update the class of the buttons to change their style.
+ * @param elem tracks the current button element chosen.
```

Comment:
Include the type in braces before the parameter name like in the formatting example. Both `document.getElementById("interests");` and `document.createElement(...)` return an `Element` object. 
https://developer.mozilla.org/en-US/docs/Web/API/element

Though in those cases they may return the subtype HTMLElement. Either way, include it in the comment.
`@param {Element elem} ...` or `@param {HTMLElement elem} ...`

Reply:
Fixed